components/styled/icon/src/organisms/icon.tsx

```diff
@@ -0,0 +1,337 @@
+import { IconType, IconSize, IconStatus, IconDirection } from '@ltht-react/styles'
+import { FC, HTMLAttributes } from 'react'
+import {
+  IconDefinition,
+  faBullseye,
+  faCalendar,
+  faCircle,
+  faSquare,
+  faCaretSquareDown,
+  faCheck,
+  faCheckSquare,
+  faChevronCircleUp,
+  faChevronCircleDown,
+  faChevronCircleLeft,
+  faChevronCircleRight,
+  faChevronUp,
+  faChevronDown,
+  faChevronLeft,
+  faChevronRight,
+  faColumns,
+  faCommentDots,
+  faTimes,
+  faDotCircle,
+  faEdit,
+  faEllipsisH,
+  faEllipsisV,
+  faExclamationTriangle,
+  faExternalLinkAlt,
+  faFilter,
+  faFolderPlus,
+  faGripHorizontal,
+  faBars,
+  faInfoCircle,
+  faKeyboard,
+  faLayerGroup,
+  faListUl,
+  faPause,
+  faPlay,
+  faPlus,
+  faSearch,
+  faSync,
+  faSquareFull,
+  faStar,
+  faStop,
+  faTable,
+  faToggleOn,
+  faToggleOff,
+  faTrashAlt,
+  faUser,
+} from '@fortawesome/free-solid-svg-icons'
+import { Transform } from '@fortawesome/fontawesome-svg-core'
+import CounterIcon from '../molecules/counter-icon'
+import StyledIcon, { IconAnimation } from '../atoms/styled-icon'
+
+const calculateChevronCircleIcon = (direction: IconDirection): IconDefinition => {
+  switch (direction) {
+    default:
+    case 'up':
+      return faChevronCircleUp
+    case 'down':
+      return faChevronCircleDown
+    case 'left':
+      return faChevronCircleLeft
+    case 'right':
+      return faChevronCircleRight
+  }
+}
+
+const calculateChevronIcon = (direction: IconDirection): IconDefinition => {
```

Comment:
Looks like these two methods are identical between this file and icon-button.tsx ?

Reply:
Not sure where you are referring to...